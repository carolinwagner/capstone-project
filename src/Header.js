import React from 'react'
import styled from 'styled-components/macro'

export default function Header() {
  return (
    <>
      <StyledHeader>Hello Bylaws</StyledHeader>
      <p>
        Jeder gemeinnützige Verein benötigt gemäß § 57 des Bürgerlichen
        Gesetzbuches eine Satzung.
      </p>
      <p>
        Die folgenden Fragen müssen im Voraus geklärt werden, damit du dir eine
        passende Satzung individuell zusammenbauen kannst.
      </p>
    </>
  )
}

const StyledHeader = styled.h1`
  font-family: 'Playfair Display', serif;
  font-size: 45px;
  color: var(--lightblue);
`
