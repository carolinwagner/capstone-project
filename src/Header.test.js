import React from 'react'
import { render } from '@testing-library/react'
import Header from './Header'

describe('Header', () => {
  it('shows the main headline', () => {
    const { getByText } = render(<Header />)
    expect(getByText('Hello Bylaws')).toBeInTheDocument()
  })
})

describe('Header', () => {
  it('renders Header correctly', () => {
    const container = render(<Header />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
