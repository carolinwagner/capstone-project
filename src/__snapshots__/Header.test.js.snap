// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders Header correctly 1`] = `undefined`;
